main.js

```diff
@@ -27,7 +27,7 @@ $.ajax({
         }
 
 				if (module.github) {
-						row += "<td class='progress first'><a href='https://travis-ci.org/" + module.github + "'><img src='https://api.travis-ci.org/" + module.github + ".svg' /></a></td>";
+						row += "<td class='progress first'><a href='https://travis-ci.com/" + module.github + "'><img src='https://api.travis-ci.com/" + module.github + ".svg' /></a></td>";
```

Comment:
Travis redirects to `/github/` these days, so we can skip that bounce:

```suggestion
						row += "<td class='progress first'><a href='https://travis-ci.com/github/" + module.github + "'><img src='https://api.travis-ci.com/" + module.github + ".svg' /></a></td>";
```